src/components/sidebar/sidebar.test.js

```diff
@@ -9,7 +10,7 @@ const mockProps = {
 
 describe('ShowMenuButton', () => {
   it('renders without crashing', () => {
-    const wrapper = setup.shallow(ShowMenuButton, mockProps);
+    const wrapper = setup.mount(<ShowMenuButton {...mockProps} />);
```

Comment:
How come this needs to be mounted now?

Reply:
It doesn't really, except that the child SVG component was showing as 0% coverage in the test coverage report and lowering the total coverage score, so by changing it to mount, it resolves this without adding pointless extra tests for the SVG icons. I figured it didn't much matter either way so this way seemed the easiest.

Incidentally before you ask, the SVG image needs to be a JS component instead of a regular SVG because imports need to work in the JS component context and we can't count on consumers having the webpack-inline-svg Babel transform installed.

Reply:
Have you investigated the use of snapshots?

Reply:
Not especially - I'm fairly ambivalent on them but usually try to avoid them because of their tendency to fail on the tiniest change. I don't have strong feelings either way though